assets/js/app.js

```diff
@@ -58,7 +58,20 @@ let csrfToken = document
 
 let liveSocket = new LiveSocket("/live", Socket, {
   params: { _csrf_token: csrfToken },
-  hooks: { ...Hooks, ...DotcomHooks }
+  hooks: { ...Hooks, ...DotcomHooks },
+  dom: {
+    onBeforeElUpdated(from, to) {
+      // Clone open state of <details>
```

Comment:
Just to refresh my memory: The point of this is because the `<.accordion />` component doesn't remember its open/closed state if its summary changes? Can we add a comment to that effect (I know there's a comment, but I mean something that explains more clearly what the consequences of not doing this are).

For instance,

```suggestion
      // By default the <.accordion /> component from metro will not remember its
      // open/closed state if the summary changes. This creates some surprising
      // behavior in the mode-selector on the trip planner, where each time you
      // change your selected mode, the menu closes.
      //
      // This code forces the `<details>` tag (what the accordion uses under the
      // hood) to remember its open/closed state to avoid that bug.
```

Reply:
Yep! It's not so much specific to Metro's accordion but it is to the HTML `<details>` open/closed state.